### DISCUSSION TITLE
"loca" - a typo?

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#321] [Q&A]
It occurs twice in the manual, but perhaps you mean "locl"?

**Comment by alerque** [comment]
No, [loca](https://learn.microsoft.com/en-us/typography/opentype/spec/loca) is part of the OpenType spec.

**Reply by jsbien** [reply]
Thanks! Are you sure it is used in the manual in this meaning?

**Reply by alerque** [reply]
Nope, only that it makes sense to see in the context of a font with lots of OpenType features.

**Comment by kenmcd** [comment]
> It occurs twice in the manual, but perhaps you mean "locl"?

Yes.
Those are discussing the feature tag `locl` not the table `loca`.
https://learn.microsoft.com/en-us/typography/opentype/spec/features_ko#tag-locl

**Comment by psb1558** [comment]
A mistake. Thanks for catching it--I have corrected it now.

**Reply by jsbien** [reply]
Glad I could help. BTW, you seem to use aalt, which is not mentioned in the manual.

**Reply by psb1558** [reply]
I don't think it's possible to document aalt usefully, since the number of variants in it is enormous, and it is likely to change from release to release. Also, users don't generally (and  shouldn't, because of the instability) use aalt directly; instead it's used by apps like InDesign to construct a glyph palette.

**Comment by jsbien** [comment]
OK
